MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -144,6 +144,8 @@ public partial class MediaEncoder : IMediaEncoder, IDisposable
         /// <inheritdoc />
         public bool IsVaapiDeviceSupportVulkanDrmInterop => _isVaapiDeviceSupportVulkanDrmInterop;
 
+        public bool IsJellyfinFfmpeg => SupportsFilter("alphasrc");
```

Comment:
I don't think `IsJellyfinFfmpeg` is a good name for this since it only tests for `alphasrc`

I don't know what `alphasrc` is, but I assume other ffmpeg builds can provide it

Reply:
>  but I assume other ffmpeg builds can provide it

Very very unlikely because this is our own custom filter. We use this filter with our subtitle burn-in pipeline but this filter is almost useless with its own. If other people decide to port this filter and not port all of our patches then this is totally on them and not us.

Reply:
I changed the name of it and added a lengthy document to this value. The assumption of having `alphasrc` means is Jellyfin ffmpeg is already widely used in the whole transcoding pipeline and there is not bug report regarding this assumption. We should be fine to keep doing it this way.